group_vars/location_fesev/networks.yml

```diff
@@ -36,14 +37,17 @@ networks:
 location__channel_assignments_11a_standard__to_merge:
   fesev-ap1: 36-40
   fesev-ap2: 40-40
+  fesev-ap4: 60-40
 
 location__channel_assignments_11g_standard__to_merge:
   fesev-ap1: 1-20
   fesev-ap2: 6-20
   fesev-ap3: 13-20
+  fesev-ap4: 11-20
```

Comment:
I would not use 11, if you already use 13?

Reply:
I  didnt wanted to touch the other APs. In the way they're installed they shoudn't interfere. But i have to update them soonish anyway, so will shift the channels